python/pyspark/mllib/evaluation.py

```diff
@@ -74,29 +80,31 @@ def __init__(self, scoreAndLabels):
         if numCol == 3:
             schema.add("weight", DoubleType(), False)
         df = sql_ctx.createDataFrame(scoreAndLabels, schema=schema)
-        java_class = sc._jvm.org.apache.spark.mllib.evaluation.BinaryClassificationMetrics
+        java_class = (
+            sc._jvm.org.apache.spark.mllib.evaluation.BinaryClassificationMetrics  # type: ignore[attr-defined]
+        )
         java_model = java_class(df._jdf)
         super(BinaryClassificationMetrics, self).__init__(java_model)
 
-    @property
+    @property  # type: ignore[misc]
```

Comment:
I met `Decorated property not supported  [misc]` for checking mypy. As mypy's issue [#1362](https://github.com/python/mypy/issues/1362), I think we should ignore this.